apps/kyberswap-interface/src/pages/Earns/utils/position.ts

```diff
@@ -392,3 +392,11 @@ export const getPositionLiquidity = async ({
 
   return
 }
+
+export const checkEarlyPosition = (position: ParsedPosition) => {
```

Comment:
if position have reward in the first 5mins, should we display the it instead of only check the time?

Reply:
@viet-nv yes, but the logic checking reward was on the component directly, this function is only to check if the position is in first 5mins
![Uploading Screenshot 2025-08-12 at 16.16.42.png…]()